crawl-ref/source/traps.cc

```diff
@@ -1162,18 +1162,19 @@ void do_trap_effects()
                 simple_god_message(" reveals an alarm trap just before you would have tripped it.");
                 return;
             }
-            mprf("With a horrendous wail, an alarm goes off!");
+            mpr("With a horrendous wail, an alarm goes off!");
             fake_noisy(40, you.pos());
             you.sentinel_mark(true);
             break;
 
         case TRAP_TELEPORT:
         {
-            string msg = make_stringf("%s and a teleportation trap spontaneously manifests!",
```

Comment:
The commit message states that a make_stringf() is replaced with a "std::string +", but this is the only make_stringf() in the commit, and, while its line is wrapped, it's still a make_stringf() after the change. I'm guessing this is an oversight?

Reply:
Sorry. I thought about changing that line, but must have decided against it.

Replacing that line with a + would have saved a line, but I can imagine someone finding the make_stringf() format easier to read. I was trying to avoid "personal preference" stuff here.

I've removed the reference from the PR description. I've left the commit message as I don't know how well it would work if I tried to change it.

Reply:
No problem. I've applied it, and edited the commit message. Thanks.